package-res/resources/web/pentaho/data/filter/Or.js

```diff
@@ -50,12 +50,16 @@ define([
    *     ]
    *   });
    *
-   *
-   *  var sales12k = new IsIn("sales", [12000]);
-   *  var inStock = new IsEqual("inStock", true);
-   *  var combination1 = new Or([sales12k, inStock]);
-   *  var data1 = combination1.apply(data); //data1.getValue(0, 0) === ["A", "B", "C"]
+   *   var sales12k = new IsIn("sales", [12000]);
+   *   var inStock = new IsEqual("inStock", true);
+   *   var combination1 = new Or([sales12k, inStock]);
+   *   var data1 = combination1.apply(data); //data1.getValue(0, 0) === ["A", "B", "C"]
```

Comment:
No. data.getValue(0, 0) returns a single value. You meant:

/*
data.getValue(0, 0) === "B"
...
data.getValue(4, 0) === "G"
*/